SpatialGDK/Documentation/content/how-to-use-schema.md

```diff
@@ -6,45 +6,57 @@
 
-* **Generate schema (Full Scan)** 
+* **Full Scan** 
 
-    To generate schema for all classes* in your project:<br/>
+    To generate schema for all classes in your project:<br/>
 
-    In the Unreal Editor, on the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons), open the **Schema** drop-down menu and select **Schema (Full Scan)**.<br/> You must select **Schema (Full Scan)** the first time you generate schema for a project. 
+    In the Unreal Editor, on the [GDK toolbar]({{urlRoot}}/content/toolbars#buttons), open the **Schema** drop-down menu and select **Schema (Full Scan)**.<br/> You must select **Schema (Full Scan)** the first time you generate schema for a project. 
     <br/> ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button-full-scan.png)<br/>
     _Image: In the GDK toolbar in the Unreal Editor, select **Schema (Full Scan)**_<br/>
-    <br/>When you select **Schema (Full Scan)**, the GDK automatically iterates through all classes* in your project to generate the schema files, and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase). <br/>
+    <br/>When you select **Schema (Full Scan)**, the GDK automatically iterates through all classes in your project to generate the schema files, and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase). <br/>
 
-    Run a full scan the first time you generate schema for your project, and whenever you need to generate schema for classes* that are not currently loaded by the Editor.<br/>For example: You need to select **Schema (Full Scan)** if you didn't generate schema after adding a new Blueprint to your game, and that Blueprint is no longer open in the Editor.<br/><br/>
+    Run a full scan the first time you generate schema for your project, and whenever you need to generate schema for classes that are not currently loaded by the Editor.<br/>For example: You need to select **Schema (Full Scan)** if you didn't generate schema after adding a new Blueprint to your game, and that Blueprint is no longer open in the Editor.<br/><br/>
 
-* **Generate Schema (Iterative)**
+* **Iterative**
 
-    To generate schema for classes* that are currently loaded by the Editor: <br/>
+    To generate schema for classes that are currently loaded by the Editor: <br/>
 
-    Select **Schema** in the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes* that are currently loaded by the Editor, generates the schema files and updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).<br/>
+    Select **Schema** in the [GDK toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes that are currently loaded by the Editor, generates the schema files and updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).<br/>
```

Comment:
Is "automatically" needed here? I think it's redundant with the more detailed explanation.

Reply:
Also "classes(*)" because it's not all classes loaded - just the ones which have replicated properties or RPCs, right? (or no?)